### ISSUE TITLE
2.5.8 regression - untransformed output with expressjs

### ISSUE THREAD

**Issue by derduher** [mde/ejs#362]
@mde I just upgraded from 2.5.7 to 2.5.8 and I'm not using readfile synchronously - just through expressjs. Although this looks similar to #355 I am not directly invoking any ejs functions.

I'm still seeing the template not getting filled out.
given a template of `<title><|= locals.foo |></title>` I get `<title>bar</title>` in 2.5.7 and `<title><|= locals.foo |></title>` in 2.5.8 (untransformed output)
```
  .get('/test', function (req, res, next) {
    let tmplParams = {foo: 'bar'}
    let promise = new Promise((resolve, reject) => res.render(path.resolve('./index-test.ejs'), tmplParams, (err, html) => {
      if (err) {
        reject(err)
      } else {
        resolve({status: tmplParams.status, html})
      }
    }))
    promise.then(tmpl => send(res, tmpl))
  })
```

**Comment by RyanZim** [comment]
Your code example doesn't show you setting the custom delimiter; perhaps that's the issue?

**Comment by derduher** [comment]
@RyanZim I am, but to speed things up I'll create a test repo.

**Comment by derduher** [comment]
@RyanZim super simple test case with 2.5.8
1. clone https://github.com/derduher/ejs-test
2. run `node index.js`
3. `curl localhost:3000/test` outputs `<title><|= locals.title |></title>`
4. `npm install ejs@2.5.7` and repeat 2-3 above with the following output resulting in `<title>titleValue</title>`

**Comment by derduher** [comment]
for ref. I am on node 8.11.1 and npm 5.8.0

**Comment by mde** [comment]
Excellent, I'll take a look. But a good place to start might be removing that wrapping Promise code. I may be totally missing something, but I can't see anything practical it accomplishes. It just makes things more complicated.

**Comment by mde** [comment]
Oh, I think this is actually a duplicate of #224 and #355. The different versions of Express have different ways of passing opts to template rendering, and none of them seem to be well documented. I'll take a look at this tonight.

**Comment by derduher** [comment]
@mde in this super simple example the promise does not accomplish much, you are correct. I wasn't sure if it was related to the issue or not so I left it in. In my server code I'm using it in conjunction with an async call to get values to populate the template. 
Thanks for looking at this. Let me know if there is any way I can help.

**Comment by mde** [comment]
Fixed and pushed to NPM, v2.5.9.

**Comment by derduher** [comment]
verified! thanks!
